espflash/resources/README.md

```diff
@@ -1,7 +1,9 @@
 # Espflash Resources
 
-The bootloaders have been built by hand from ESP-IDF, using the `release/v5.1` branch:
-https://github.com/espressif/esp-idf/tree/release/v5.1
+The listed bootloaders from `espressif/esp-idf` were built with `release/v5.4` at commit `3ad3632`, using default settings:
+https://github.com/espressif/esp-idf/tree/release/v5.4
 
-The ESP32s flasher stubs are taken from the **esp-rs/esp-flasher-stub** repository:
-https://github.com/esp-rs/esp-flasher-stub/releases/latest
+For now, `esp-hal` uses MMU page size as `0x10000` (64k) therefore the flash size has to be changed to 64MB.
```

Comment:
Do we want to have this duplicated? Id either remove it from this readme or from [`espflash/resources/bootloaders/README.md`](https://github.com/esp-rs/espflash/blob/main/espflash/resources/bootloaders/README.md) as we already forgot to update one of the two places, and it will happen again in the future.

Reply:
Not sure what you mean, I've deleted that file as I'm not sure why it was added in the first place

Reply:
Oh! I didn't noticed you deleted the `bootloaders/README`, sorry Im just blind!